src/api/java/com/minecolonies/api/configuration/Configurations.java

```diff
@@ -189,6 +189,9 @@
         @Config.Comment("Should citizen name tags be rendered?")
         public boolean alwaysRenderNameTag = true;
 
+        @Config.Comment("Child growth modifier, default on avg they take about 20min to grow. Setting to 2 = 2x as fast. Default value: 1")
+        public int growthModifier = 1;
```

Comment:
could we make he default time = 1h and then maybe make the default growth modifier = 3 to allow people to set it in both directions (maybe make it even 10h and set default 30)

Reply:
This should also be a double not an int.
If servers want it to take longer they should be able to set it to 0.5 for say 2 hours

Reply:
👍